src/layouts/react-spacing.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server'; // tslint:disable-line:no-submodule-imports
+
+export function testSpacing(body: string): ReadonlyArray<string> {
+  const regexTextBeforeElement = /[0-9a-zA-Z\.\,\?\!\:\)]+\<[0-9a-zA-Z]+.+?\>/g;
+  const regexTextAfterElement = /\<\/[0-9a-zA-Z]\>[0-9a-zA-Z\(\[]+/g;
+
+  const missingSpacesBefore = body.match(regexTextBeforeElement) || [];
+  const missingSpacesAfter = body.match(regexTextAfterElement) || [];
+
+  return [...missingSpacesBefore, ...missingSpacesAfter];
+}
+
+describe(testSpacing, () => {
+  it('correctly checks whether any spaces are missing before or after elements', () => {
```

Comment:
I suggest renaming `testSpacing` to `spacesMissingAroundInlineElements` or similar? :)

Reply:
Sounds good :)